frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -336,9 +336,14 @@ public static function createNomination(
                         );
                     }
                 }
-                if (self::hasDuplicates($contents['tags'])) {
+
+                $duplicatedTags = self::hasDuplicates($contents['tags']);
```

Comment:
La función ahora no sólo indica si el contenido tiene duplicados, sino que también obtendrá esa lista de contenido duplicado, ¿te parece si le cambiamos el nombre por `getDuplicatedTags`?
```suggestion
                $duplicatedTags = self::getDuplicatedTags($contents['tags']);
```